server/middlewares/authorization.js

```diff
@@ -0,0 +1,22 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import Util from '../helpers/util';
+
+
+dotenv.config();
+
+const util = new Util();
+
+// eslint-disable-next-line consistent-return
+export const authorized = async (req, res, next) => {
```

Comment:
Good work on the authorization, but
I would like to suggest maybe use a name that is giving more meaning like `authorizationCheck, checkAuthorization`, or others.

Reply:
That's what I was suggesting if we can have a common middleware for authorization because the application has to use the same authorization way.